pkg/x509.go

```diff
@@ -59,15 +60,22 @@ const TLSCertificateValidityInDays = 365
 // SigningCertificateValidityInDays holds the number of days issued signing certificates are valid
 const SigningCertificateValidityInDays = 365
 
-// Certificate validity in days according to Nuts RFC003
+// RFC003ValidityInDays is the number of days a certificate is valid according to Nuts RFC003
 const RFC003ValidityInDays = 4
 
-// RFC003CertificateProfile is a x509.CertificateProfile according to RFC003: signing the JWT bearer token
-var RFC003CertificateProfile = CertificateProfile{
+// RFC003SigningCertificateProfile is a x509.CertificateProfile according to RFC003: signing the JWT bearer token
+var RFC003SigningCertificateProfile = CertificateProfile{
```

Comment:
I don't like exported vars since they can be altered. Unless that's intended, maybe wrap it in a function which is called when the struct is needed (`RFC003SigningCertificateProfile()` or `GetRFC003SigningCertificateProfile()`)

Reply:
it has more to do with the fact that RFC is an abbreviation and therefore required capitals in 'normal' text. I lower-cased all of them.